.github/workflows/schema.yml

```diff
@@ -0,0 +1,32 @@
+name: Update JSON schema 
+
+on:
+  push:
+		paths:
+		- ../../schemas/'schema.json'
```

Comment:
Should this be `schema.json`? or `scripts/pydanticJSONgenerator.py`?

Reply:
You can add a `workflow_dispatch` here for testing manually.

Reply:
Changed to scripts/pydanticJSONgenerator.py